pkg/scheduler/framework/interface.go

```diff
@@ -307,10 +307,16 @@ type QueueSortPlugin interface {
 }
 
 // EnqueueExtensions is an optional interface that plugins can implement to efficiently
-// move unschedulable Pods in internal scheduling queues.
+// move unschedulable Pods in internal scheduling queues. Only plugins
+// impacting scheduling will take effect, other implemented plugins are useless and
+// harmless.
+// When setting up a new scheduler, it will load events and convert them to gvks
+// to build dynamic event handlers with.
```

Comment:
```suggestion
// to dynamically build event handlers.
```
or simply remove this line, as it is a redundant information.